cmake/features.cmake

```
@@ -167,8 +167,11 @@ ie_option(ENABLE_OV_ONNX_FRONTEND "Enable ONNX FrontEnd" ${PYTHONINTERP_FOUND})
 ie_option(ENABLE_OV_PADDLE_FRONTEND "Enable PaddlePaddle FrontEnd" ON)
 ie_option(ENABLE_OV_IR_FRONTEND "Enable IR FrontEnd" ON)
 ie_option(ENABLE_OV_TF_FRONTEND "Enable TensorFlow FrontEnd" ON)
+ie_option(ENABLE_OV_TF_LITE_FRONTEND "Enable TensorFlow Lite FrontEnd" ON)
```

```suggestion
ie_dependent_option(ENABLE_OV_TF_LITE_FRONTEND "Enable TensorFlow Lite FrontEnd" ON "ENABLE_OV_TF_FRONTEND" OFF)
```
With this change we don't have to additionally check `if(ENABLE_OV_TF_FRONTEND)` for TF Lite

---

not relevant anymore since TF and TF LITE are independent now